### DISCUSSION TITLE
Access custom headers inside the auth hook function

### DISCUSSION THREAD

**Discussion by chira37** [supabase/supabase#23615] [Questions]
I need to access to custom header and add that attribute to the jwt, is there any way to access to the custom header inside the auth hook postgres function

```
create or replace function public.custom_access_token_hook(event jsonb)
returns jsonb
language plpgsql
as $$
declare
  -- Insert variables here
begin
  -- Insert logic here
  return event;
end;
$$;
```

do we have access to the http headers inside that event?

**Comment by GaryAustin1** [comment]
No.  Normally those are available on database operations because of PostgREST handling them.

**Comment by Jokero** [comment]
Is there a working solution now maybe?
